sqlmesh/core/context.py

```diff
@@ -777,6 +777,7 @@ def table_diff(
         Returns:
             The TableDiff object containing schema and summary differences.
         """
+        source_env_name, target_env_name = "SOURCE", "TARGET"
```

Comment:
ditto: why do we need these default values? It might be a little confusing to a user. It might be even more confusing even there are real environments named "source" and / or "target".

Reply:
good point